ui/src/main/resources/Meeting/MeetingSheet.xml

```diff
@@ -107,10 +107,9 @@ $xwiki.jsx.use('Meeting.MeetingSheet')
   #set ($discard = $mailmsg.addPart('text/html', "$message &lt;br&gt; &lt;a href=$link&gt; $services.localization.render('contrib.meeting.notification.link') &lt;/a&gt;"))
   #getVCalendarContent($doc, $vCalendar)
   #set ($discard = $mailmsg.addPart('text/calendar;method=CANCEL', $vCalendar, {"headers" : {"Content-Class" : "urn:content-classes:calendarmessage"}}))
-  #set ($discard = $mailmsg.send())
-
-  #if($mailmsg.getErrors().size() == 0)
-    $services.localization.render('contrib.meeting.notification.success')
+  #set ($result = $services.mailsender.send([$mailmsg], "database"))
+  #if($result.isSent())
+    {{success}}$services.localization.render('contrib.meeting.notification.success'){{/success}}
```

Comment:
This is far from enough. How do you report errors? Please check the doc of the mail sender module to have exemples of how to do proper error reporting. Also check the javadoc for isSent() since it doesn't do at all what you think it does... I'll change the method name to isProcessed() to be less ambiguous.